lib/glimesh_web/live/support_modal/support_modal.ex

```diff
@@ -0,0 +1,60 @@
+defmodule GlimeshWeb.SupportModal do
+  use GlimeshWeb, :live_view
+
+  alias Glimesh.Accounts
+
+  @impl true
+  def mount(_params, %{"streamer" => streamer, "user" => nil} = session, socket) do
+    if session["locale"], do: Gettext.put_locale(session["locale"])
+
+    channel = Glimesh.ChannelLookups.get_channel_for_user(streamer)
+    can_receive_payments = Accounts.can_receive_payments?(streamer)
+
+    {:ok,
```

Comment:
Should be able to combine these two function's assigns since they are so similar.